[IGNORE] add project documentation (#1479)
perses/perses: docs/project.md

# Project

> A project can be defined as a workspace. It's the place where user will spend most of their time, creating dashboard and

Reviewer: ```suggestion
A project can be defined as a workspace. It's the place where user will spend most of their time, creating dashboards, variables and
```

Reviewer 2: +user**s**

> datasource.

Reviewer: ```suggestion
datasources.
```

**You need to own a project if you want to create a dashboard**

> Creating a project is simple as providing the name:

Reviewer 2: ```suggestion
Creating a project is as simple as providing its name:
```
